.repo/sample-list-generator/src/prepare-chrome-types.ts

```diff
@@ -4,12 +4,19 @@ import fs from 'fs/promises';
 import { ExtensionApiMap } from './types';
 import { ReflectionKind } from 'typedoc';
 
+// Bucket used to store processed types data
+const STORAGE_BUCKET = process.env.STORAGE_BUCKET;
```

Comment:
Should we mention this in the readme?

Reply:
Yeah. Once we figure out how we're going to handle this going forward I'll update it.